### ISSUE TITLE
Build failure on `nightly_linux_clang_libcxx` with `rmf_fastrtps_shared_cpp`

### ISSUE THREAD

**Issue by Yadunund** [ros2/rmw_fastrtps#688]
Console log from failure: https://ci.ros2.org/view/nightly/job/nightly_linux_clang_libcxx/1492/console

```bash
14:25:48 In file included from /home/jenkins-agent/workspace/nightly_linux_clang_libcxx/ws/src/ros2/rmw_fastrtps/rmw_fastrtps_shared_cpp/src/custom_participant_info.cpp:26:
14:25:48 In file included from /home/jenkins-agent/workspace/nightly_linux_clang_libcxx/ws/src/ros2/rmw_fastrtps/rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_participant_info.hpp:46:
14:25:48 /home/jenkins-agent/workspace/nightly_linux_clang_libcxx/ws/src/ros2/rmw_fastrtps/rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_event_info.hpp:81:42: error: expected ';' at end of declaration list
14:25:48   bool inconsistent_topic_changed_{false}
14:25:48                                          ^
14:25:48                                          ;
14:25:48 1 error generated.
14:25:48 gmake[2]: *** [CMakeFiles/rmw_fastrtps_shared_cpp.dir/build.make:76: CMakeFiles/rmw_fastrtps_shared_cpp.dir/src/custom_participant_info.cpp.o] Error 1
14:25:48 gmake[1]: *** [CMakeFiles/Makefile2:207: CMakeFiles/rmw_fastrtps_shared_cpp.dir/all] Error 2
14:25:48 gmake: *** [Makefile:146: all] Error 2
14:25:48 ---
```

**Comment by fujitatomoya** [comment]
@Yadunund can we just fix this with https://github.com/ros2/rmw_fastrtps/pull/689?

**Comment by clalancette** [comment]
This particular error was solved by #689, so closing.
